src/nuclei/client/main.py

```diff
@@ -207,28 +269,36 @@ def get_endpoints(self, app: str) -> List[str]:
         ConnectionError:
             Application not available
         TypeError:
-            Wrong type for `app` argument
+            Wrong type for `app` or `version` argument
         ValueError:
-            Wrong value for `app` argument
+            Wrong value for `app` or `version` argument
```

Comment:
We are missing type check for version and value error checks in the code.

Reply:
This is a propagating error, its in self.get_url.